plugins/taoqtiimage/lang/af.js

```diff
@@ -0,0 +1,4 @@
+CKEDITOR.plugins.setLang( 'insertQtiImage', 'af', {
```

Comment:
No need to add all languages, we have translations only for `de.js`, `fr.js`, `nl.js`
And for other languages will be used default

Reply:
I think in this case, we need to add `en` that is the default, otherwise will appear that key `insertQtiImage` that is worse than `Insert Qti Image`

Reply:
I think you are right, should be also `en.js`

Reply:
done